### PR TITLE
Update use cases

### PR DESCRIPTION
**Pull request by yihuididi**

### CODE REVIEW

**Review by rithiklenin — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -340,104 +348,345 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
    Use case ends.
 
 **Use case: UC2 - Add a client**
 
 **MSS**
 
-1. User requests to add client
-1. Prudy adds the client
+1. User requests to add client by entering the details (name, phone, email, address, and optional tags).
```
Comment (ckaayy) [inline docs/DeveloperGuide.md:371]: the use case numbering seems to all be 1, is that the desired numbering?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -340,104 +348,345 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case: UC4 - List policies of a client**
 
 **MSS**
 
-1. User <u>list clients (UC1)</u> to look for index of client
-1. User requests to add a policy to a specific client in the list
-1. Prudy adds the policy under the client
+1. User <u>list clients (UC1)</u> to look for index of client.
+1. User request to list policies of client.
+1. Prudy shows a list of the client's policies.
```
Comment (rithiklenin) [inline docs/DeveloperGuide.md:431]: this part too but the DG seems fine

**Review by ckaayy — APPROVED** [review]
lgtm

**Pull request merged**